### ISSUE TITLE
Overfitting? 

### ISSUE THREAD

**Issue by ozanpkr** [alankbi/detecto#84]
**Describe the bug**
    I used the same dataset while training mmdetection models (RetinaNet,FasterRCNNresnet50, or SSD). I got well result on my test set and validation set. But when I used the detecto , a validation loss increased continuously and I got 41.2% AP  after training process.
    How can I solve that? Have you any suggestions?

**Code and Data**

![train](https://user-images.githubusercontent.com/59290581/118238657-7b854d00-b4a1-11eb-9579-a74fc4d98f19.JPG)

**Environment:**
 - OS: windows
 - Python version: 3.8
 - Detecto version:1.2.1
 - torch version: 1.7.1
 - torchvision version: 0.8.2

**Additional context**

![out](https://user-images.githubusercontent.com/59290581/118237938-7d9adc00-b4a0-11eb-916c-cc8f254a2214.png)

**Comment by alankbi** [comment]
Could you try using different parameters, such as the values passed to fit or the batch size for the DataLoader? Outside of that, I'd also recommend experimenting with different transforms to maybe augment your dataset. 

Overall, if it still has trouble performing well, it might be too complicated of a dataset for Detecto or require additional fine-tuning that you might want to do with PyTorch.

**Comment by ozanpkr** [comment]
> Could you try using different parameters, such as the values passed to fit or the batch size for the DataLoader? Outside of that, I'd also recommend experimenting with different transforms to maybe augment your dataset.
> 
> Overall, if it still has trouble performing well, it might be too complicated of a dataset for Detecto or require additional fine-tuning that you might want to do with PyTorch.

Thanks for a quick reply. I will change parameters(batch, initial lr, etc.) and then try.
